src/components/LeafletMap.js

```diff
@@ -233,6 +233,30 @@ class LeafletMap extends Component {
     return false;
   }
 
+  toFixedNoRounding = function(number, n) {
```

Comment:
You should call this function something along the lines of `setSignificantFeaturesToCoordinate` or something similar. Don't worry about long names, webpack will always minify them on production builds 🙂